services/frontend-service/src/legacy-ui/ReleaseDialog.tsx

```diff
@@ -366,17 +352,6 @@ const ReleaseEnvironment: VFC<{
     );
     const currentlyDeployedVersion = overview.environments[environmentName].applications[applicationName]?.version;
 
-    const deleteQueue: CartAction = useMemo(
-        () => ({
-            deleteQueue: {
-                application: applicationName,
-                environment: environmentName,
-                currentlyDeployedVersion: currentlyDeployedVersion,
-            },
-        }),
-        [applicationName, environmentName, currentlyDeployedVersion]
-    );
-
     const queuedVersion = overview.environments[environmentName].applications[applicationName]?.queuedVersion || 0;
```

Comment:
Question: Should we remove the queuedVersion for here as well?

Reply:
The story only included removing the button and not the message